core/storage/feedback/gae_models.py

```diff
@@ -41,7 +41,6 @@
 
 class FeedbackThreadModel(base_models.BaseModel):
     """Threads for each exploration.
-
```

Comment:
Done, that was actually a copy paste error. Apparently copying from github skips empty lines.